backend/register_lambda.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+
+dynamodb = boto3.client('dynamodb')
+
+def lambda_handler(event, context):
+    params = event['queryStringParameters']
+    email = params['email']
+    dropbox_oauth_token = params['dropbox_oauth_token']
+
+    dynamodb.put_item(
+        TableName='Emails',
+        Item={
+            'email': {'S': email},
+            'dropbox_oauth_token': {'S': dropbox_oauth_token},
+        }
+    )
+
+    response = {
```

Comment:
minor thing here, since it is a passthrough, if we do fail for whatever reason itll just show a generic 500 since we aren't formatting any response types - obviously not an mvp thing though

Reply:
Yep - will fix in a follow up diff